### ISSUE TITLE
1.5.1 changes to Util.formatNum break many features

### ISSUE THREAD

**Issue by desean1625** [Leaflet/Leaflet#6873]
```js
var line = L.polyline([[0,-7.993322e-10],[10,20],[20,30]]);
L.geoJson(line.toGeoJSON());//errors on NaN
L.Util.formatNum(-7.993322e-10); //Returns NaN
```

Please publish a patch to revert of this commit.

https://github.com/Leaflet/Leaflet/pull/6670
**Environment**
- Leaflet version: 1.5.1
- Browser (with version):
- OS/Platform (with version):

**Comment by cherniavskii** [comment]
Hi @desean1625 
I'm going to publish a patch later this week.
Sorry for the delay!

**Comment by cherniavskii** [comment]
Released in 1.6.0
